packages/fluent-theme/src/icons/InfoSmallIcon.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export function InfoSmallIcon(props: Readonly<{ readonly className?: string }>) {
```

Comment:
Should we add `memo`?

Also, prefer `export default` unless we are exporting at root `index.ts`.

```suggestion
export default memo(function InfoSmallIcon(props: Readonly<{ readonly className?: string }>) {
```